cypress/e2e/store.cy.js

```diff
@@ -0,0 +1,30 @@
+// cypress/e2e/filters.cy.js
+
+import FiltersPage from "../support/page-objects/filters.page";
+import StorePage from "../support/page-objects/store-page";
+
+describe('Write a review for the searched product', () => {
+    beforeEach(() => {
+        cy.visit('/search?q=shoes')
+        FiltersPage.waitForResults()
+        FiltersPage.selectFirstProduct()
+    })
+
+    it('store page displayed for the selected product', () => {
+        StorePage.storePageDisplayed().should('exist');
+    })
+
+    it('select to write a review for the product ', () => {
+        StorePage.selectWriteReviewOpton();
+        StorePage.elements.reviewTitle().should('exist');
+
+        // Fill in the review form
+        const reviewTitle = 'Amazing Shoes!';
+        const reviewContent = 'These shoes are comfortable, stylish, and affordable. Highly recommend!';
+        const name = 'John';
+        const email = 'test@example.com';
```

Comment:
Why did you chose to put these into variables?

Reply:
I chose to assign these values to variables (reviewTitle, reviewContent, name, and email) to improve readability and maintainability of the test. By extracting them into constants:

It’s easier to update the content in one place if needed, rather than searching through the test logic.
It makes the test steps cleaner and helps highlight what data is being used in the review form.